journal/original/03-chapter-2.adoc

```diff
@@ -77,7 +101,10 @@ as the descendants of the first inhabitants of this land of North America,
 whom our forefathers found here after they had crossed the great water.
 Desiring that the chain of our friendship may be kept bright, we bid you farewell.
 
-Signed by forty-four Friends.
+[.signed-section-context-open]
```

Comment:
This needs to be a `[.signed-section-signature]` instead of a `[.signed-section-context-open]` (I've mixed these up too. The easiest way to spot an error of this kind is to click the new button that Jared made on the editor that renders a mock-pdf in your browser... then scan through the chapter. If a signature is not in all caps, or is on the left (instead of the right) side of the page... then you it probably got tagged wrong. 👍

Reply:
Thank you!! I will use it, I haven't yet.

Reply:
Thanks for all these little reminders, btw.